.github/workflows/int-and-connected-test-run.yml

```
@@ -108,103 +130,48 @@ jobs:
-          helm_gcloud_sqlproxy_chart_version: 0.19.13
-          helm_oidc_proxy_chart_version: 0.0.44
-          sa_b64_credentials: ${{ secrets.SA_B64_CREDENTIALS }}
-      - name: Fetch gitHash for deployed integration version
-        id: configuration
-        run: |
-          git_hash=$(git rev-parse --short HEAD)
-          echo "git_hash=${git_hash}" >> $GITHUB_OUTPUT
-          echo "Latest git hash for this branch: $git_hash"
-      - name: Wait for deployment to come back online
-        uses: broadinstitute/datarepo-actions/actions/wait-for-deployment@0.74.0
-        timeout-minutes: 20
-        env:
-          DESIRED_GITHASH: ${{ steps.configuration.outputs.git_hash }}
-          DEPLOYMENT_TYPE: 'api'
-      - name: Run test runner smoke tests via Gradle
-        uses: broadinstitute/datarepo-actions/actions/main@0.74.0
-        env:
-          # unset the GitHub Action default JAVA_HOME to run with JDK 17
-          JAVA_HOME:
-          # output plain logs instead of json
-          TDR_LOG_APPENDER: 'Console-Standard'
-        with:
-          actions_subcommand: 'gradletestrunnersmoketest'
-          sa_b64_credentials: ${{ secrets.SA_B64_CREDENTIALS }}
-      - name: Run integration tests via Gradle
-        uses: broadinstitute/datarepo-actions/actions/main@0.74.0
+      - name: Run integration tests
         env:
           # required for azure tests
           AZURE_CREDENTIALS_APPLICATIONID: 0e29ec36-04e8-44d5-ae7c-50dc15135571
           AZURE_CREDENTIALS_HOMETENANTID: fad90753-2022-4456-9b0a-c7e5b934e408
           AZURE_CREDENTIALS_SECRET: ${{ secrets.AZURE_CREDENTIALS_SECRET }}
-          # unset the GitHub Action default JAVA_HOME to run with JDK 17
-          JAVA_HOME:
+          # required for integration tests
+          GOOGLE_APPLICATION_CREDENTIALS: jade-dev-account.json
+          GOOGLE_SA_CERT: jade-dev-account.pem
+          IT_JADE_API_URL: http://localhost:8080
+          # postgres connection details
+          PGHOST: 127.0.0.1
+          PGPASSWORD: postgres
+          # required for integration tests
+          RBS_CLIENT_CREDENTIAL_FILE_PATH: rbs-tools-sa.json
           # output plain logs instead of json
           TDR_LOG_APPENDER: 'Console-Standard'
-        with:
-          actions_subcommand: 'gradleinttest'
-          pgport: ${{ job.services.postgres.ports[5432] }}
-          test_to_run: 'testIntegration'
-          sa_b64_credentials: ${{ secrets.SA_B64_CREDENTIALS }}
-      - name: Clean state lock from used Namespace on API deploy
-        if: always()
-        uses: broadinstitute/datarepo-actions/actions/main@0.74.0
-        with:
-          actions_subcommand: 'k8_checknamespace_clean'
-          sa_b64_credentials: ${{ secrets.SA_B64_CREDENTIALS }}
-      - name: Clean whitelisted Runner IP
+        run: |
+          # wait for postgres to be ready
+          until pg_isready -h ${PGHOST} -p 5432; do
+            sleep 1
+          done
```

Each `pg_isready` invocation waits 3s for a connection. Instead of a retry with sleep, you can use `-t 10` or `-t 30` to wait 10 or 30 seconds instead of waiting forever. Also `-p 5432` is unnecessary as 5432 is the default
```suggestion
         pg_isready -h ${PGHOST} -t 10
```